qualtran/surface_code/azure_cost_model.ipynb

```diff
@@ -108,7 +108,7 @@
     "\n",
     "Where $a$ and $b$ are constants that depend on the gate set and the approximation protocol. Table 1 of [Kliuchnikov et al](https://arxiv.org/abs/2203.10064) gives estimates for those constants for different combinations of gate sets and protocols.\n",
     "\n",
-    "In the paper, they use $a=0.53, b=5.3$"
+    "In the paper, they use $a=0.53, b=4.86$"
```

Comment:
What do you think about the wording:
"Though [Beverland et al](https://arxiv.org/abs/2211.07629) includes slightly different coefficient values, the rotation approximation model used by Azure cost model is using $a=0.53 and b=4.86$ from Table 1 of the [Kliuchnikov et al](https://arxiv.org/abs/2203.10064)."

Reply:
sounds good to me!

Reply:
@NoureldinYosri do you want to weigh in before I push another commit.

Reply:
we chatted offline. your wording is perfect